src/utils/getCalendarMonthWidth.js

```diff
@@ -0,0 +1,5 @@
+const CALENDAR_MONTH_PADDING = 9;
```

Comment:
probs. the `DAY_PICKER_PADDING is also separate. This is the padding on each month (and between months as well).